### PR TITLE
Feature supported doris

### PR DESCRIPTION
**Pull request by wtt40122**
#133 
Log support for Doris writing and querying

### CODE REVIEW

File: ozhera-log/log-manager/src/main/java/com/xiaomi/mone/log/manager/service/extension/store/DefaultStoreExtensionService.java [hunk 1]
```diff
@@ -62,41 +64,58 @@ public class DefaultStoreExtensionService implements StoreExtensionService {
     @Resource
     private MilogLogstoreDao logStoreDao;
 
+    @Resource
+    private DorisLogStorageService dorisLogStorageService;
```
Comment (goodjava) [inline ozhera-log/log-manager/src/main/java/com/xiaomi/mone/log/manager/service/extension/store/DefaultStoreExtensionService.java:68]: The excuse needs to replace this in the future.

**Review by goodjava — APPROVED** [review]
+1

File: ozhera-log/log-manager/src/main/java/com/xiaomi/mone/log/manager/service/extension/store/DefaultStoreExtensionService.java (hunk 1 above)
Comment (wtt40122) [reply ozhera-log/log-manager/src/main/java/com/xiaomi/mone/log/manager/service/extension/store/DefaultStoreExtensionService.java:68]: This is a progress and we look forward to increasing it in the future

Comment (wtt40122) [reply ozhera-log/log-manager/src/main/java/com/xiaomi/mone/log/manager/service/extension/store/DefaultStoreExtensionService.java:68]: > The excuse needs to replace this in the future.

This is a progress and we look forward to increasing it in the future

**Review by wodiwudi — APPROVED** [review]
+LGTM

**Pull request merged**